### PR TITLE
Exporting nightwatch interfaces

### PR DESCRIPTION
**Pull request by emadow**
Exporting nightwatch types so they can be imported without having them as globals.

cc: @mohsen1

### CODE REVIEW

**Comment by mohsen1** [comment]
🔒

File: types/nightwatch/nightwatch-tests.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
+import { NightwatchAPI } from 'nightwatch';
```
Comment (mohsen1) [inline types/nightwatch/nightwatch-tests.ts]: relative import here

Comment (mohsen1) [reply types/nightwatch/nightwatch-tests.ts]: I was WRONG!

**Comment by mohsen1** [comment]
+1

/cc @andy-ms

**Comment by Schlesiger** [comment]
Thanks! It's nice to have some additional eyes on this definition.

**Comment by Schlesiger** [comment]
Actually, do you mind just giving an example on how the Nightwatch definition could be extended if it is not global? For example, I also wrote the following definition for [nightwatch-vrt](https://github.com/Crunch-io/nightwatch-vrt): [index.d.ts](https://github.com/Crunch-io/nightwatch-vrt/blob/master/index.d.ts).

**Comment by andy-ms** [comment]
@Schlesiger 
```ts
declare module "nightwatch" {
  export interface Foo { }
}
```
This code must appear inside of a module (a file with `import` / `export).

**Pull request merged**